StudyTaskApi.UnitTests/ApiTests.cs

```diff
@@ -22,9 +22,9 @@ public void SetUp()
             }
         }
```

Comment:
```suggestion
        }

```
I suggest adding empty lines between methods